### DISCUSSION TITLE
Error: failed to create new program: Operation no longer supported, see https://go.deta.dev/migrate/

### DISCUSSION THREAD

**Discussion by juanmaluna21** [deta/space-docs#39] [Q&A] [answered]
Hi guys, does anyone know how to solve this problem?
I try to create my micro with these commands:
- deta new
- deta new --python first_micro
but I get always the same result:
this one: Error: failed to create new program: Operation no longer supported, see https://go.deta.dev/migrate/
I have been looking for a solution but found nothing so far.
I am in a mac big sur, not sure if my pc is the problem.
I appreciate your help!

**Comment by sudo-subg** [comment, marked as answer]
Following the upgrade on February 14, Deta Cloud has been merged with [Deta Space](https://deta.space/developers). As a result, you can no longer create new Projects or Micros on Deta Cloud. Existing Projects and Micros on Deta Cloud will continue to run, uninterrupted with support until May 1, 2023. So, you should migrate your Cloud Micros to Space before May 1, 2023.
Check out https://go.deta.dev/migrate/deta-new to learn more about it.

**Comment by hari-silvers** [comment]
Check out the Space-docs on working with the project I also had the same issue the documentation described about three steps and I have moved the code to the space but code show error on importing pyodbc module which has downloaded while pushing the code

**Reply by sudo-subg** [reply]
hey @silverking-hari, please share the error message. And if possible a repo with minimal source code to reproduce this including the `Spacefile` would be helpful.

**Comment by hotlizimu** [comment]
What's wrong with my deta app?
When I use space new
It shows ❗ Error: failed to create a project, developer mode needs to be enabled to build apps
